Library/Homebrew/software_spec.rb

```diff
@@ -439,13 +439,30 @@ def checksums
       # Sort non-MacOS tags below MacOS tags.
       "0.#{tag}"
     end
-    sha256s = tags.reverse.map do |tag|
+    tags.reverse.map do |tag|
       {
-        collector[tag][:checksum] => tag,
-        cellar: collector[tag][:cellar],
+        "tag"    => tag,
+        "digest" => collector[tag][:checksum],
+        "cellar" => collector[tag][:cellar],
       }
     end
-    { sha256: sha256s }
+  end
+
+  def generate_sha256_line(tag, digest, cellar)
+    sha256_line = "sha256 \"#{digest}\" => :#{tag}"
+    if cellar.present?
```

Comment:
```suggestion
    sha256_line = "sha256 \"#{digest}\" => :#{tag}"
    return sha256_line if cellar.blank?

```

Could let you reduce the intention here.